tests/XamlParserTests/Impl/Fixtures.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using XamlX;
+using XamlX.Parsers;
+using Xunit;
+
+namespace XamlParserTests.Impl
+{
+
+    public class GuiLabsFixture : IDisposable
+    {
+        public GuiLabsFixture()
+        {
+            XamlParser.UseXDocumentParser = false;
```

Comment:
Probably we could simplify the code a bit if we just store the XAML parser under test (either `XDocument` or `GuiLabs`) in a field like `XamlParser` instead of dancing with a boolean flag. E.g. if one installs `XamlX.Parser.GuiLabs` they definitely want to use `GuiLabsXamlParser`